docs/asm-gke-terraform/README.md

```diff
@@ -0,0 +1,105 @@
+## Managed Anthos Service Mesh on GKE with Terraform 
+
+This is a general setup of a single cluster with Anthos Service Mesh using the native [GKE Hub Terraform resource](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/gke_hub_feature) for the examples in the `docs` folder.
```

Comment:
Let's replace `Hub` by `Fleet` as the recommended approach in our docs (b/222286504).

Even if the TF resource/API is still `google_gke_hub_*` on a doc/wording perspective, it's recommended to use `Fleet` instead of  `Hub`.

Reply:
gotcha, good point